src/functions/advanced-functions/event.php

```diff
@@ -33,8 +33,10 @@
 	 * - EventShowMap bool - Set to true to embed the map in the Event view.
 	 * - EventCost string - Default cost of the Event.
 	 * - EventURL string - Link to the Event Website or Third-Party page
+ 	 * - FeaturedImage string - URL or ID of a featured image
```

Comment:
This is just an observation about the docblock in general—and not a comment on your change specifically—but, for cases like this, we should _probably_ be following this convention:

```php
/**
 * Create an event. ...
 *
 * @param array $args {
 *     @type string $EventStartDate Start date of event (required).
 *     @type string $EventEndDate End date of event (required).
 *     ...
 * }
 *
 * @otherTags
 */
```

I am pretty sure this jives nicely with WP Parser (thinking docs site). See: [make.wordpress.org "Inline Documentation Standards (parameters that are arrays)."](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-1-parameters-that-are-arrays)

Reply:
I confirm, proper doc block structure, while painful to type, has that structure.

Also: all doc-block lines should end in a full-stop, a question mark or an exclamation point.